smac/initial_design/initial_design.py

```diff
@@ -98,6 +98,8 @@ def __init__(
 
     def select_configurations(self) -> List[Configuration]:
         """Selects the initial configurations."""
+        if self.init_budget == 0:
```

Comment:
If we don't have an initial budget, the next configuration to sample directly comes from the unfitted (bc no data) GP/RF, right?

Reply:
Not 100% sure but I think so